charts/refinery/values.yaml

```diff
@@ -228,6 +267,96 @@ config:
+  # arrive for traces that were active during Stress Relief will respect those
+  # decisions.
+  #
+  # The measurement of stress is a lagging indicator and is highly dependent on
+  # Refinery configuration and scaling. Other configuration values should be well
+  # tuned first, before adjusting the Stress Relief Activation parameters.
+  StressRelief:
+    # Mode is a string indicating how to use Stress Relief. Options are:
+    # - "never" means that Stress Relief will never activate
+    # - "monitor" is the recommended setting, and means that Stress Relief will monitor
+    #     the status of refinery and activate according to the levels set below.
+    # - "always" means that Stress Relief is always on, which may be useful in an
+    #     emergency situation.
+    Mode: "never"
+
+    # ActivationLevel is the stress_level (from 0-100) at which Stress Relief is triggered.
+    ActivationLevel: 75
+
+    # DeactivationLevel is the stress_level (from 0-100) at which Stress Relief is
+    # turned off (subject to MinimumActivationDuration). Under normal circumstances,
+    # it should be well below ActivationLevel to avoid oscillations.
+    DeactivationLevel: 25
+
+    # StressSamplingRate is the sampling rate to use when Stress Relief is
+    # activated. All new traces will be deterministically sampled at this rate based
+    # only on the traceID.
+    StressSamplingRate: 100
+
+    # MinimumActivationDuration is the minimum time that stress relief will stay
+    # enabled, once activated. This prevents oscillations.
+    MinimumActivationDuration: 10s
+
+    # MinimumStartupDuration is used when switching into Monitor mode.
+    # When stress monitoring is enabled, it will start up in stressed mode for a
+    # at least this amount of time to try to make sure that Refinery can handle the load
+    # before it begins processing it in earnest. This is to help address the
+    # problem of trying to bring a new node into an already-overloaded
+    # cluster. If this duration is 0, Refinery will not start in stressed mode.
+    # This can provide faster startup at the possible cost of startup instability.
+    MinimumStartupDuration: 3s
+
+  # Sample Cache Configuration controls the sample cache used to retain information about trace
+  # status after the sampling decision has been made.
+  SampleCacheConfig:
+
+    # Type controls the type of sample cache used.
+    # "legacy" is a strategy where both keep and drop decisions are stored in a circular buffer that is
+    # 5x the size of the trace cache. This is Refinery's original sample cache strategy.
+    # "cuckoo" is a strategy where dropped traces are preserved in a "Cuckoo Filter", which can remember
+    # a much larger number of dropped traces, leaving capacity to retain a much larger number of kept traces.
+    # It is also more configurable. The cuckoo filter is recommended for most installations.
+    Type: "legacy"
```

Comment:
similarly here -- could we set this to cuckoo in the chart?

Reply:
Same reason as before. Since we didn't use it as a default value in refinery then the same rule would apply here. Defaulting to cuckoo here is the same as defaulting to cuckoo in refinery.